app/src/main/java/com/codesoom/assignment/repository/MemoryTaskRepository.java

```diff
@@ -0,0 +1,39 @@
+package com.codesoom.assignment.repository;
+
+import com.codesoom.assignment.models.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class MemoryTaskRepository {
+    private List<Task> tasks = new ArrayList<>();
+    private Long sequence = 0L;
+
+    public Task save(Task task){
+        task.setId(sequence++);
+        tasks.add(task);
+        return task;
+    }
+
+    public Optional<Task> findById(Long taskId){
+        return tasks.stream().filter(task-> task.getId().equals(taskId)).findAny();
```

Comment:
읽는 사람(코드 리뷰어, 저)이 읽기 쉽도록 한 줄에 `.`이 하나만 오도록 적절히 개행해 봅시다.
한 줄에 `.`이 하나씩 오는 건 강력한 규칙은 아니고, 보통 이렇게 하면 논리적인 단계 하나가 한 줄에 표현되기 때문입니다.

```suggestion
        return tasks.stream()
           .filter(task-> task.getId().equals(taskId))
           .findAny();
```

Reply:
넵 훨씬 보기 좋은 것 같습니다. 수정했습니다.